### PR TITLE
added rest swagger

### PR DESCRIPTION
**Pull request by 0xBACE**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,19 +11,24 @@ Django==4.2
...
+django-rest-swagger==2.2.0
 django-templated-mail==1.1.1
 djangorestframework==3.14.0
 djangorestframework-simplejwt==5.2.2
 djoser==2.2.0a0
 drf-jwt==1.19.2
+drf-yasg==1.21.5
 gunicorn==20.1.0
 idna==2.8
+inflection==0.5.1
```
Comment (Henriknn01) [inline requirements.txt:23]: remove this

File: requirements.txt [hunk 2]
```diff
@@ -11,19 +11,24 @@ Django==4.2
...
 install==1.3.5
 itypes==1.2.0
 Jinja2==3.1.2
 Markdown==3.4.1
 MarkupSafe==2.1.2
 oauthlib==3.2.2
+openapi-codec==1.3.2
```
Comment (Henriknn01) [inline requirements.txt:30]: remove this

File: requirements.txt [hunk 3]
```diff
@@ -11,19 +11,24 @@ Django==4.2
...
+packaging==23.1
```
Comment (Henriknn01) [inline requirements.txt:31]: remove this

File: requirements.txt [hunk 4]
```diff
@@ -32,10 +37,13 @@ python3-openid==3.2.0
 pytz==2022.7
 requests==2.28.2
 requests-oauthlib==1.3.1
+ruamel.yaml==0.17.26
+ruamel.yaml.clib==0.2.7
```
Comment (Henriknn01) [inline requirements.txt:41]: remove this and the above + simeplejson

File: requirements.txt [hunk 5]
```diff
@@ -32,10 +37,13 @@ python3-openid==3.2.0
...
+simplejson==3.19.1
 six==1.16.0
 social-auth-app-django==5.2.0
 social-auth-core==4.4.1
 sqlparse==0.4.3
+tzdata==2022.7
```
Comment (Henriknn01) [inline requirements.txt:47]: remove

File: requirements.txt (hunk 3 above)
Comment (0xBACE) [reply requirements.txt:30]: this is needed

**Review by Henriknn01 — APPROVED** [review]
lgtm

**Pull request merged**